static/docs/command-reference/import.md

```diff
@@ -3,7 +3,7 @@
 Download or copy file or directory from any <abbr>DVC project</abbr> in a Git
 repository (e.g. hosted on GitHub) into the <abbr>workspace</abbr>, and track
 changes in this [external dependency](/doc/user-guide/external-dependencies).
-Creates a DVC-file.
+Creates a special DVC-file a.k.a _import stage_.
```

Comment:
let's avoid complicating headers. It's already too long.

Reply:
Reverting...